css/style.css

```diff
@@ -0,0 +1,92 @@
+
+/*#fadeImage{
+  background: white;
+}*/
+
+
+#container{
+  display: none;
+  max-width: 960px;
+  margin: auto;
+}
+
+body{
+  background: url("space4.gif");
+}
+
+#page-header{
+  border:0;
+  box-sizing: border-box;
+  clear: both;
+  display: block;
+  float: left;
+  margin: auto;
+  padding: 2% 8% 2% 8%;
```

Comment:
If the first two values are the same as the second two, you can omit them. So it would look like `padding: 2% 8%`